src/languages/en.js

```diff
@@ -16,6 +16,7 @@ export default {
         password: 'Password',
         profile: 'Profile',
         payments: 'Payments',
+        payPalMe: 'PayPal.me',
```

Comment:
Does this need to be included in translations? I think that proper nouns like brand names will be the same in all translations, and therefore can be excluded from the translation files. Right?

Reply:
If not, this is missing from the spanish translation.

Reply:
Hm good question. Yeah I think we probably don't need to translate this

Reply:
Might have missed this along the many comments, but I still don't think this needs to be translated.